packages/ethereum/tsconfig.hardhat.json

```diff
@@ -7,5 +7,5 @@
     "rootDir": "."
   },
   "include": ["type-extension.ts", "hardhat.config.ts", "tasks", "test", "deploy", "src"],
-  "references": [{ "path": "../utils" }]
+  "references": [{ "path": "../utils" }, { "path": "../core" }]
```

Comment:
Why does the `ethereum` package need a reference to `core`?

Reply:
It's the `hardhat-ethereum` package. Only used by `hardhat` script, such as `hardhat faucet` or `hardhat network`.

Within the `hardhat` scripts, there are some references to `core`